library/rockylinux

```diff
@@ -2,18 +2,18 @@ Maintainers: The Rocky Linux Foundation <infrastructure@rockylinux.org> (@rocky-
              Neil Hanlon <neil@resf.org> (@neilhanlon)
 GitRepo: https://github.com/rocky-linux/sig-cloud-instance-images.git
 
-Tags: 9.0.20220720, 9, 9.0
-GitFetch: refs/heads/Rocky-9.0.20220720-Base-x86_64
-GitCommit: e3f3c1cbc89d705fad4cb99e5786a4dc7178116c
-arm64v8-GitFetch: refs/heads/Rocky-9.0.20220720-Base-aarch64
-arm64v8-GitCommit: 78129b594f5318c80fd24b0b0d25dfb35125c9d6
+Tags: 9.1.20221119, 9, 9.1
+GitFetch: refs/heads/Rocky-9.1.20221119-Minimal-x86_64
+GitCommit: ec9fb969a767fe74591f9ce39e09b44659a56cf4
+arm64v8-GitFetch: refs/heads/Rocky-9.1.20221119-Minimal-aarch64
+arm64v8-GitCommit: 624fe1e35788615384eaf94345cc8dcd3919e594
 Architectures: amd64, arm64v8
 
-Tags: 9.0.20220720-minimal, 9-minimal, 9.0-minimal 
-GitFetch: refs/heads/Rocky-9.0.20220720-Minimal-x86_64
-GitCommit: ac4d852f696bb2cc96c55f4a54b1fb7d41f5df4f
-arm64v8-GitFetch: refs/heads/Rocky-9.0.20220720-Minimal-aarch64
-arm64v8-GitCommit: bbea3587002f91144f3195b8b0372f05257fd6f5
+Tags: 9.1.20221119-minimal, 9-minimal, 9.1-minimal
+GitFetch: refs/heads/Rocky-9.1.20221119-Minimal-x86_64
+GitCommit: ec9fb969a767fe74591f9ce39e09b44659a56cf4
+arm64v8-GitFetch: refs/heads/Rocky-9.1.20221119-Minimal-aarch64
+arm64v8-GitCommit: 624fe1e35788615384eaf94345cc8dcd3919e594
 Architectures: amd64, arm64v8
 
 Tags: 8.6.20227707, 8.6, 8
```

Comment:
I think you probably meant to drop/update this entry instead of just adding a new one?  (See the PR CI failures :eyes:)

Reply:
I think my latest commit should address this.

My intention is to: build 8.7 and 9.1 images, and move the respective 8 & 9 tags to point to the newer ones.